client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/pattern-large-preview.scss

```diff
@@ -14,6 +16,10 @@
 
 	.pattern-assembler__pattern-panel-list--is-open & {
 		margin-inline-start: 365px;
+
+		@media ( max-width: $break-wide ) {
+			margin-inline-start: 360px;
+		}
```

Comment:
It might be better to use the `break-wide` mixin for consistency as we usually define the styles for the small screen first, and then use `break-x` mixins to deal with the large screens. WDYT?

```scss
margin-inline-start: 360px;

@include break-wide {
  margin-inline-start: 365px;
}
```

Reply:
+1. I also prefer the mobile-first approach 🙂

Reply:
Sure +1